spyder/plugins/ipythonconsole.py

```diff
@@ -907,7 +909,7 @@ def write_to_stdin(self, line):
 
     @Slot()
     @Slot(bool)

```

Comment:
Please add two more signatures below this line

    @Slot(object)
    @Slot(bool, object)

to cover all possible entries of `create_new_client`.